.circleci/config.yml

```diff
@@ -201,4 +238,4 @@ workflows:
                   family: "app"
                   cluster-name: "Graphelier-cluster"
                   service-name: "Graphelier-service"
-                  container-image-name-updates: "container=graphelier-app,image-and-tag=${AWS_ECR_URL}/graphelier-app:${CIRCLE_SHA1},container=graphelier-service,image-and-tag=${AWS_ECR_URL}/graphelier-service:${CIRCLE_SHA1},container=mongo,image-and-tag=${AWS_ECR_URL}/mongo:${CIRCLE_SHA1},container=graphelier-news,image-and-tag=${AWS_ECR_URL}/graphelier-news:${CIRCLE_SHA1},container=news-db,image-and-tag=${AWS_ECR_URL}/news-db:${CIRCLE_SHA1}"
+                  container-image-name-updates: "container=graphelier-app,image-and-tag=${AWS_ECR_URL}/graphelier-app:${CIRCLE_SHA1},container=graphelier-service,image-and-tag=${AWS_ECR_URL}/graphelier-service:${CIRCLE_SHA1},container=service-db,image-and-tag=${AWS_ECR_URL}/mongo:${CIRCLE_SHA1},container=graphelier-news,image-and-tag=${AWS_ECR_URL}/graphelier-news:${CIRCLE_SHA1},container=news-db,image-and-tag=${AWS_ECR_URL}/news-db:${CIRCLE_SHA1}"
```

Comment:
why aren't you changing the second instance of `mongo here` ?

Reply:
because im renaming the container from mongo to service-db